gems/pending/Scvmm/miq_hyperv_disk.rb

```diff
@@ -150,7 +150,7 @@ def bread(start_sector, number_sectors)
       encoded_data = @parser.output_to_attribute(run_correct_powershell(read_script))
       t2           = Time.now.getlocal
       buffer       = ""
-      Base64.decode64(encoded_data).split(' ').each { |c| buffer += c.to_i.chr }
+      Base64.decode64(encoded_data).split(' ').each { |c| buffer << c.to_i.chr }
```

Comment:
@roliveri the output of the powershell script is individual Base64 encoded binary bytes that are space-separated.
We've got to put them back together again.